site/_layouts/home.html

```diff
@@ -2,68 +2,77 @@
     A spiritual successor to TodoMVC
   </h2>
-  <p class="f5 f4-ns mt0 silver lh-copy mw8">
+  <p class="f5 f4-ns mt0 mid-gray lh-copy mw8">
     TodoMVC has helped thousands of developers select
     an MV* framework for their JavaScript applications. However, the web ecosystem
     has evolved in the past few years allowing us to build powerful applications using
     modern browser capabilities.
   </p>
 
-  <p class="f5 f4-ns silver lh-copy mw8">
+  <p class="f5 f4-ns mid-gray lh-copy mw8">
     To provide developers with examples, we collected a list of Hacker News clients built with a number of popular JavaScript frameworks and libraries.
-    Each implementation is a complete <a class="b gray link hover-orange" href="https://developers.google.com/web/progressive-web-apps/">Progressive Web App</a> that utilizes
+    Each implementation is a complete <a class="b dark-gray link hover-orange" href="https://developers.google.com/web/progressive-web-apps/">Progressive Web App</a> that utilizes
     different progressive technologies to provide a fast, reliable and engaging experience.
   </p>
 </div>
 
 {% assign sortedApps = site.apps | sort: 'weight' %}
 
 {% for app in sortedApps %}
-  <article class="h6-l bt b--black-10 mid-gray lh-copy-ns ph2 ph5-m pv4 pv5-ns avenir {% cycle 'bg-white', 'bg-near-white' %}" id={{ app.title }}>
+  <article class="h6-l bt b--black-10 dark-gray lh-copy-ns ph2 ph5-m pv4 pv5-ns avenir {% cycle 'bg-white', 'bg-near-white' %}" id={{ app.title }}>
     <div class="flex flex-row justify-center center">
       <div class="flex h-inherit items-center mr7-l">
         <div class="flex-column">
-          <a class="flex items-center justify-center justify-start-l mb4 no-underline" href={{ app.framework-link }}>
+          <a class="flex items-center justify-center justify-start-l mb4 no-underline link" href={{ app.framework-link }}>
             <img src={{ app.logo | relative_url}} class="grow-large pa1 ba b--black-10 br-100 h2 w2 h3-ns w3-ns" alt={{ app.framework }}>
-            <h1 class="mid-gray ma0 f3 f2-ns fw4 pl2 pl3-ns tracked">{{ app.title }}</h1>
+            <h1 class="dark-gray ma0 f3 f2-ns fw4 pl2 pl3-ns tracked">{{ app.title }}</h1>
           </a>
 
           <dl class="lh-title mt0 mb4 tc tl-l">
             <dt class="f6 f4-ns b">Lighthouse</dt>
             <dd class="ml0 f5 fw5 f4-m f3-l pt1">
-              <div class="flex items-center justify-center justify-start-l silver">
+              <div class="flex items-center justify-center justify-start-l mid-gray">
                 {{ app.lighthouse }}
-                <a class="flex grow-large ml1 ml2-l link-svg" href={{ app.lighthouse-link }} target="_blank" rel="noopener">{% include link-icon.svg %}</a>
+                <a class="flex grow-large ml1 ml2-l link-svg link" href={{ app.lighthouse-link }} target="_blank" rel="noopener">
+                    <span class="visuallyhidden">View Lighthouse results</span>
```

Comment:
Instead of doing this I'd suggest using an `aria-label` on your anchor element. Otherwise the screen reader focus moves to the offscreen hidden item and then moves back to the definition list.

e.g.

`<a aria-label="View Lighthouse results" class="flex grow-large ml1 ml2-l link-svg link" href={{ app.lighthouse-link }} target="_blank" rel="noopener">`